app/js/main.js

```diff
@@ -111,10 +114,24 @@ function openIndex() {
     });
 }
 
+/**
+ * Confirm prompt when animator is closed.
+ */
+win.on("close", function() {
+    confirmSet(closeAnimator, "", "Are you sure you to exit Boats Animator?");
```

Comment:
Rewrite as so:
`confirmSet(win.close, true, "Are you sure you to exit Boats Animator?");`